<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tableconfigurationsettingswindow</class>
 <widget class="QDialog" name="tableconfigurationsettingswindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Table Configuration</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxTableMode">
   <property name="geometry">
    <rect>
     <x>202</x>
     <y>20</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelTableModeTitle">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Table Mode:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCurrTableMode">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxAiDiff">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelAiDiffTitle">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>AI Difficulty:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCurrAiDiff">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTableLightingTitle">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Table Lighting (RGB):</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCurrTableLighting">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSliderTableLighting">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>170</y>
     <width>160</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>16777215</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelTableAirSpeedTitle">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Table Air Speed (CFM):</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCurrTableAirSpeed">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>68</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSliderTableAirSpeed">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>230</y>
     <width>160</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>tableconfigurationsettingswindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>tableconfigurationsettingswindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
